.gitignore

```diff
@@ -27,8 +27,8 @@ artifacts/
 # JUnit Test Reports
 junit.xml
 tests-*.xml
-logs/
-bin/
+/logs/
+/bin/
```

Comment:
Before this was ignoring the bin generated for contracts, but I saw that there are already files committed for those so clarified this for consistency